src/key/verification/start.rs

```diff
@@ -415,10 +415,20 @@ mod tests {
             content: key_verification_start_content,
         };
 
-        assert_eq!(
-            to_string(&key_verification_start).unwrap(),
-            r#"{"content":{"from_device":"123","transaction_id":"456","method":"m.sas.v1","key_agreement_protocols":["curve25519"],"hashes":["sha256"],"message_authentication_codes":["hkdf-hmac-sha256"],"short_authentication_string":["decimal"]},"type":"m.key.verification.start"}"#
-        );
+        let json_data = json!({
+            "content":{
```

Comment:
Please add spaces after the `"key":` part here and below.

```suggestion
            "content": {
```

Reply:
alright, i have made these changes, i hope i got everything.

Reply:
If you haven't, that can still be fixed later.